src/app/util/PhotoUtil.ts

```diff
@@ -0,0 +1,18 @@
+import { getCollectionSize } from 'app/util/PhotoCollectionResolver'
+import { PhotoActionController } from 'app/controller/PhotoActionController'
+import { PhotoCollection } from 'app/state/StateTypes'
+import toaster from 'app/Toaster'
+import { msg } from 'common/i18n/i18n'
+import { formatNumber } from 'common/util/TextUtil'
+
+export function movePhotosToTrash(photos: PhotoCollection | null | undefined, controller: PhotoActionController) {
```

Comment:
Please don't introduce a new module here. Instead you can add an optional `showToast` parameter to `PhotoActionController` and put the toast code to the `defaultPhotoActionController`.